modules/installation-network-upi.adoc

```diff
@@ -4,7 +4,20 @@
 // * installing/installing_vsphere/installing-vsphere.adoc
 
 [id="installation-network-upi-{context}"]
-= UPI network topology requirements
+= UPI networking requirements
```

Comment:
Personally I prefer to merge "UPI networking requirements" and "Connectivity between machines" as one section, their context is closely relative. Is that reasonable?

Reply:
Sure! I moved the table to this section. Is this better?